src/app/jobs/components/job-detail/job-detail.component.ts

```diff
@@ -53,18 +57,21 @@ export class JobDetailComponent implements OnInit, OnDestroy {
           selectNotNull(getJob, {jobId}),
           map((j) => {
             this.ninjaService.getPredict(jobId).subscribe((jobPrediction: PathwayResponse) => {
-              const jobs = JSON.parse(localStorage.getItem('jobs'));
-              const jobIndex = jobs.findIndex(((job) => job.id === parseInt(jobId, 10)));
-              jobs[jobIndex].state = jobPrediction.status;
+              jobPrediction.type = 'Pathway prediction';
```

Comment:
It seems odd that the type declaration needs to be defined in the component. Isn't it the responsibility of the service to set this?

Reply:
It should be, but right now I dont have it in the service and I need it, maybe I can create cards, and work in the backend a little bit, what do you think? And I can include the method and the type :)